packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/CameraUtils.java

```diff
@@ -169,6 +121,28 @@ public static Messages.PlatformDeviceOrientation orientationToPigeon(
     return Messages.PlatformDeviceOrientation.PORTRAIT_UP;
   }
 
+  /**
+   * Converts a PlatformDeviceOrientation from Pigeon to DeviceOrientation from PlatformChannel.
+   *
+   * @param orientation A PlatformDeviceOrientation
+   * @return The corresponding DeviceOrientation. Defaults to PORTRAIT_UP.
+   */
+  @NonNull
+  public static PlatformChannel.DeviceOrientation orientationFromPigeon(
+      @NonNull Messages.PlatformDeviceOrientation orientation) {
+    switch (orientation) {
+      case PORTRAIT_UP:
+        return PlatformChannel.DeviceOrientation.PORTRAIT_UP;
+      case PORTRAIT_DOWN:
+        return PlatformChannel.DeviceOrientation.PORTRAIT_DOWN;
+      case LANDSCAPE_LEFT:
+        return PlatformChannel.DeviceOrientation.LANDSCAPE_LEFT;
+      case LANDSCAPE_RIGHT:
+        return PlatformChannel.DeviceOrientation.LANDSCAPE_RIGHT;
+    }
+    return PlatformChannel.DeviceOrientation.PORTRAIT_UP;
```

Comment:
My intention was to return a default value here in case there is ever a new value added to this enum, as we do not define it, so that the plugin does not break.